libsolidity/interface/Natspec.cpp

```diff
@@ -118,3 +116,26 @@ string Natspec::extractDoc(multimap<string, DocTag> const& _tags, string const&
 		value += i->second.content;
 	return value;
 }
+
+void Natspec::fillDevDocumentation(Json::Value &_json, std::multimap<std::string, DocTag> const &_tags)
+{
+	auto dev = extractDoc(_tags, "dev");
+	if (!dev.empty())
+		_json["details"] = Json::Value(dev);
+
+	auto author = extractDoc(_tags, "author");
+	if (!author.empty())
+		_json["author"] = author;
+
+	auto ret = extractDoc(_tags, "return");
+	if (!ret.empty())
+		_json["return"] = ret;
```

Comment:
I don't think `return` should be valid for the constructor.

Reply:
Oh yeah! Good catch!